libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -541,7 +541,11 @@ export class DtStackedSeriesChart implements OnDestroy, OnInit {
     series?: DtStackedSeriesChartSeries,
     node?: DtStackedSeriesChartNode,
   ): false | void {
-    return this._isNodeSelectionMode && this._toggleSelect(series, node);
+    return (
+      this._isNodeSelectionMode &&
+      !node?.disabled &&
```

Comment:
Nit: We could be a bit more explicit here as we had shifty boolean value checks in the past.
```suggestion
      node?.disabled !== false &&
```

Reply:
That way we also need to invert boolean value, so it will be `node?.disabled !== true`

Reply:
That way we need to invert boolean value, so `node?.disabled !== true`